zipkin-lens/src/translations/fr/messages.json

```diff
@@ -0,0 +1,36 @@
+{
+  "1 Result": "1 Résultat",
+  "Address": "Adresse",
+  "Annotations": "Annotations",
+  "Archive Trace": "Trace Archivée",
+  "Dependencies": "Dépendances",
+  "Depth": "Profondeur",
+  "Download JSON": "Télécharger JSON",
+  "Duration": "Durée",
+  "End Time": "Fin",
+  "Find a trace": "Trouver une trace",
+  "Limit": "Limite",
+  "None": "Aucun",
+  "Parent ID": "ID Parent",
+  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionner un critère dans la barre de recherche. Et, cliquer sur le bouton de recherche",
+  "Please select the start and end time. Then, click the search button.": "S'il vous plaît sélectionner la date de début et de fin. Et, cliquer sur le bouton de recherche",
+  "Relative Time": "Temps Relatif",
+  "Root": "Racine",
+  "Search Dependencies": "Rechercher des dépendances",
```

Comment:
Given the UI I would have said "Rechercher les dépendances" ?

Reply:
At first, I translated by " Rechercher les dépendances" but after use UI, I replaced by "Rechercher des dépendances"... 
I have doubt about use "les" ou "des" but I'm going to follow your idea